docs/rst/user_manual/layout.rst

```diff
@@ -219,22 +219,26 @@ For the explanation of this information refer to the section :ref:`log_panel`.
 
 Main Panel
 ==========
-The central window it will display the data charts, so-called *Chartbox*, that the user has configured.
-To create a new *Chartbox* go to *Edit->DisplayNewData* in the application menu.
-This button will create a new *Chartbox* where new series could be added, removed or modified.
+The central window allows multiple tabs for different views.
+Also it is displayed a collapsed menu with the possible problems that have been detected on the DDS entities.
+The tab panel allows multiple tabs generation so different views are contemplated in the same window.
+It will display the data charts, so-called *Chartbox*, that the user has configured.
+It will display a domain graph where the physical, logical and DDS entities from a domain are represented,
+focusing on the connection between endpoints through the topics, and the entities physical inheritance.
 
 .. figure:: /rst/figures/screenshots/main_panel.png
     :align: center
 
-For further information about how to create a chart refer to the section :ref:`chart_panel`.
-
 .. _chartbox_layout:
 
 Chartbox
```

Comment:
This section is a bit confusing. Chartbox seems to be the charts displayed in the chart view (it even talks about moving chartbox along the main window). The previous description on how to create a new chartbox seems more appropiate.
There is one only chart view with several chart boxes.

Reply:
The app is cap now to allow  only one chart view with multiple chartboxes. Eventually, we may support the generation of multiple chart views, so that description will be outdated.
I am keeping it until we fix that.